crates/router/src/connector/globalpay/transformers.rs

```diff
@@ -162,15 +185,10 @@ impl<F, T>
             types::PaymentsResponseData,
         >,
     ) -> Result<Self, Self::Error> {
+        let status = enums::AttemptStatus::from(item.response.status.clone());
```

Comment:
Then you can get rid of this clone and elsewhere in the module, if you've done any.